src/OpenSage.Game/Data/InstallationLocator.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Microsoft.Win32;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace OpenSage.Data
+{
+    public enum GameId
```

Comment:
Having read more of the code, re-using `OpenSage.Data.Ini.SageGame` might make sense here? It could also be moved higher in the namespace hierarchy if it's commonly used.

Reply:
Yes - let's move `SageGame` to the root of the OpenSage.Game project, in the `OpenSage` namespace.